react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+
+class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      currenButtonDisplayed: ''
```

Comment:
State should track the `id` of the "open" topic

Reply:
I cleaned up the assignment
![resubmission](https://user-images.githubusercontent.com/71291742/138394426-1ebb3d00-09d7-4e08-88ad-656afffdcefe.gif)
t